src/test/java/com/google/javascript/clutz/partial/static_method.js

```diff
@@ -0,0 +1,18 @@
+goog.provide('foo.bar.Class');
+
+/**
+ * @constructor
+ */
+foo.bar.Class = function() {
+};
+
+//!! The method param references to foo.bar.Class should be renamed, along with
```

Comment:
renamed to what?  (don't have to spell out exactly the target of it, but rather something "renamed to a dotted-module access" or whatever)

Reply:
Done